src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstNum) => {
+    counter++;
+
+    return (secondNum) => {
+      const sum = firstNum + secondNum;
+
+      return counter >= 3 && counter % 2 === 0 ? 'Bzzz... Error!' : sum;
```

Comment:
```suggestion
      return counter >= 3 && counter % 2 === 0 
      	? 'Bzzz... Error!' 
      	: sum;
```

Reply:
Yep, it's better to break lines in ternaries.